cell-filtering-igv/index.html

```diff
@@ -37,26 +34,60 @@
   return originalChrFeatures
 }
 
-  function filterAtac(scoreSelection) {
-    const trackIndex = 4 // Track index
-    const igvBrowser = window.igvBrowser
+/** Determine if feature is in genomic frame */
+function getIsFeatureInFrame(feature, igvBrowser) {
```

Comment:
Why is this function defined twice, both here and in `GenomeView`?

Reply:
This area intended is as a 0-config, rapid prototyping environment.  It's super rough -- stuff is copied and pasted, poorly formatted, etc. -- but helps iterate 5-20x faster.